### ISSUE TITLE
[A11y] Screen reader focus is not moving to last column in "Version history" table while navigating Ctrl+Alt+Right Arrow keys

### ISSUE THREAD

**Issue by keylime-unicorn** [NuGet/NuGetGallery#9472]
URL: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1745411

Environment Details:
App name: NuGet Service
#URL: [https://www.nuget.org](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fwww.nuget.org%2f&data=05%7c01%7cv-nitipi%40microsoft.com%7c9a84307e7aa744a4b24208dafa5ca2fe%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638097573052590837%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=C8Eab3i6ixAmqsOjG4AxNT36z9WZQiXPbjDDkMcS5ME%3D&reserved=0)
OS and Browser Details:
Edition Windows 11 Enterprise
OS build 22621.1105.
Chrome version:  109.0.5414.120 (Official Build) (64-bit)
Screen Reader: JAWS.

Repro Steps:
Hit the URL: https://www.nuget.org/
Navigate to "Sign in" control and hit enter.
Navigate to "Sign in with Microsoft" control and hit enter.
Search for a key term (say “json”) from search box.
Click on "Newtonsoft.json" control and hit enter.
Navigate to "Version History" control and hit enter.
Observe whether when screen reader focus on user unable to navigate last column "Version history" table with the help of Ctrl+Alt+Right Arrow keys or not.

Actual Result:
Screen reader focus is not moving to last column in "Version history" table while navigating Ctrl+Alt+Right Arrow keys

Expected Result:
Screen reader focus should move to last column in "Version history" table while navigating Ctrl+Alt+Right Arrow keys

User Impact:
Screen reader not able to understand last column "Version history" information table. 

Recommendations:
Refer [Repository of bug fixes code snippets](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2faka.ms%2faccsnippet&data=05%7c01%7cv-samhithaks%40microsoft.com%7c535b4a834ef843acdfb508da53769f91%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637914066217264155%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=xoIfLB9Iq6HVIPyUc42rS%2BlVQCt9m%2BaUokp32Kaz1nE%3D&reserved=0)

**Comment by theot12** [comment]
PR merged into dev. Verified in DEV. Marked as resolved on AzDo.

**Comment by theot12** [comment]
Waiting on confirmation from tester that this issue is resolved.
